pallets/utxo/src/lib.rs

```diff
@@ -220,6 +279,20 @@ pub mod pallet {
         CallPP(AccountId, bool, Vec<u8>),
         /// Pay to script hash
         ScriptHash(H256),
+        /// First attempt of staking.
+        /// Must assign a controller, in order to bond and validate. see pallet-staking
+        LockForStaking {
+            stash_account: AccountId,
+            controller_account: AccountId,
+            session_key: Vec<u8>,
```

Comment:
This is a very weird way of doing things. Why does the user have to provide a session key? Isn't that automatically generated from the controller key?